tools/Resources.py

```diff
@@ -180,7 +180,12 @@ def get_latest_stat_multiple(target, token, uuids, key):
             return False
 
         if response.status_code == 200:
-            return response.json()['values']
+            try:
+                return response.json()['values']
+            except json.decoder.JSONDecodeError as e:
+                print("Catching JSONDecodeError for target:", str(target), "and key:", str(key),
```

Comment:
please add spaces after the `:`

Reply:
with a comma a space is inserted automatically